esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
+    full: 'Chrome',
+      short: 'CH 53',
+      note_id: 'experimental-flag',
+      unstable: true,
+  },
+  chrome54: {
+    full: 'Chrome',
+      short: 'CH 54',
```

Comment:
Looks like I didn't check this file after extraction browsers. Need to fix it, maybe in separate PR.